src/Npgsql/NpgsqlRawCopyStream.cs

```diff
@@ -187,40 +296,43 @@ async Task<int> Read(byte[] buffer, int offset, int count, bool async)
                     throw;
                 }
 
-                switch (msg.Code) {
-                case BackendMessageCode.CopyData:
-                    _leftToReadInDataMsg = ((CopyDataMessage)msg).Length;
-                    break;
-                case BackendMessageCode.CopyDone:
-                    Expect<CommandCompleteMessage>(await _connector.ReadMessage(async), _connector);
-                    Expect<ReadyForQueryMessage>(await _connector.ReadMessage(async), _connector);
-                    _isConsumed = true;
-                    return 0;
-                default:
-                    throw _connector.UnexpectedMessageReceived(msg.Code);
+                switch (msg.Code)
```

Comment:
Undo indentation changes

Reply:
grrr... I have disabled csharp_indent_switch_labels in .editorconfig to avoid that in the future 😎